src/ckanext-gla_theme/ckanext/gla_theme/public/gla_theme.css

```
@@ -1079,4 +1079,10 @@ a.tag:hover {
 .connections img {
     display: block;
     width: 100%;
-}
\ No newline at end of file
+}
+.user-image {
+    width: 0 !important;
+    height: 0 !important;
+    visibility: hidden !important;
```

This pages suggests just setting `display: none` if you want to hide an element and remove it from the page layout:
https://developer.mozilla.org/en-US/docs/Web/CSS/visibility

```suggestion
    display: none !important;
```